src/components/Badge/README.md

```diff
@@ -5,26 +5,25 @@ A Badge component is a UI element that helps indicate the count, state, or statu
 ## Example
 
 ```jsx
-<Badge status='info'>Jazz Flute</Badge>
+<Badge status="info">Jazz Flute</Badge>
 ```
 
-
 ## Props
 
-| Prop | Type | Description |
-| --- | --- | --- |
-| className | `string` | Custom class names to be added to the component. |
-| display | `string` | Determines the CSS `display` of the component. Default `inlineBlock`. |
-| status | `string` | Changes the color of the component to the corresponding status. |
-| size | `number` | Adjust component size. |
-| white | `bool` | Applies a white style to the component. |
-
+| Prop      | Type     | Description                                                           |
+| --------- | -------- | --------------------------------------------------------------------- |
+| color     | `string` | A custom color value that can be passed in                            |
```

Comment:
Should the Readme include the `inverted` prop as well?

Reply:
g1, 👍